app/public/map.js

```diff
@@ -1,5 +1,22 @@
-/* global L */
-(function () {
+/* global L XMLHttpRequest */
+
+var nazarethNuggets = (function () { // eslint-disable-line
+
+  function requestNuggets (method, url, callback) {
+    var xhr = new XMLHttpRequest()
+    xhr.onreadystatechange = function () {
+      if (xhr.readyState === 4 && xhr.status === 200) {
+        var response = JSON.parse(xhr.responseText)
+        callback(null, response)
+      }
+      if (xhr.status === 500) {
+        callback(null, 'err')
```

Comment:
this callback is being called with error=null and result='err', which is probably not what we want, right?